gradio/interface.py

```diff
@@ -769,6 +788,20 @@ def integrate(self, comet_ml=None, wandb=None, mlflow=None) -> None:
             utils.integration_analytics(data)
 
 
+class TabbedInterface(Blocks):
+    def __init__(
+        self, interface_list: List[Interface], tab_names: Optional[List[str]] = None
+    ):
+        if tab_names is None:
+            tab_names = ["Tab {}".format(i) for i in range(len(interface_list))]
+        super().__init__()
+        with self:
+            with Tabs():
+                for (interface, tab_name) in zip(interface_list, tab_names):
+                    with TabItem(label=tab_name):
+                        interface.render_basic_interface()
```

Comment:
We could do `Interface.__init__(interface)` as well right @aliabid94?

Reply:
Didn't work for me: `TypeError: __init__() missing 1 required positional argument: 'fn'`

Reply:
Oh it actually should be `Block.__init__ ` for init to register in Context.

```
                    for component in self.input_components:
                        Block.__init__(component)
```

Reply:
No error with `Block.__init__`, but the Interfaces inside the tabs don't show up:

<img width="705" alt="image" src="https://user-images.githubusercontent.com/1778297/162216380-7de35f2c-b162-40be-8e50-851f9b4fba16.png">

So let's just stick to `render_basic_interface()`

Reply:
why can't we run Interface.render() using the Blocks render method()

Reply:
It doesn't render the component either. Let's leave it like this right now since it doesn't cause any issues and gives us control to make sure only the `Interface` kwargs that are relevant inside a `TabbedInterface` are included